pkg/config/containers.conf

```diff
@@ -1,29 +1,48 @@
 # The containers configuration file specifies all of the available configuration
-# options and command-line flags for container runtime tools like Podman & Buildah,
+# command-line options/flags for container runtime tools like Podman & Buildah,
 # but in a TOML format that can be easily modified and versioned.
 
 # Please refer to containers.conf(5) for details of all configuration options.
+# Not all container tools implement all of the options.
+# All of the options have hard coded defaults and these options will override
+# the built in defaults. Users can then override these options via the command
+# line. Container engines will read containers.conf files in up to three
+# locations in the following order:
+#  1. /usr/share/containers/containers.conf
+#  2. /etc/containers/containers.conf
+#  3. $HOME/.config/containers/containers.conf (Rootless containers ONLY)
+#  Items specified in the latter containers.conf, if they exist, override the
+# previous containers.conf # settings, or the default settings.
```

Comment:
I'm still not clear on the ordering for rootless.  If a variable is defined in all three files with different values, what does it get set to in a rootless environment?  Does the processing really go through all three files and the value from 1 would be overridden by the 2nd, and then finally by the 3rd file's value?  If so, then this is good, if not and we instead read just file 3 in a rootless environment, I think this needs to be adjusted.

Reply:
Goes through all three, and LAST one with a field get's used.

So if ~/.config/containers/containers.conf has a field in a rootless container, it wins.  If it does not, then the precedence goes
/etc/containers/containers.conf
/usr/share/containers/containers.conf
Defaults.

All of these are checked even in the case of rootless.

I look at these 4 configuration settings as being controlled by different entities.
Defaults -> Upstream
/usr/share -> distros
/etc -> administrators
$HOME-> Users
Each able to override the other.